web/app/themes/ppj/src/sass/components/header.sass

```diff
@@ -42,27 +42,102 @@
     top: 0
     width: 100%
 
-  &__site-header
-    background: linear-gradient(to bottom, $color-transparency-dark-start, $color-transparent)
+  &__nav
     box-sizing: border-box
     z-index: 1
 
-    &-content
-      @extend .l-full
-      display: flex
-      justify-content: space-between
-      margin-bottom: 0
-      margin-top: 0
-      opacity: 1
-      padding: $common-spacing
+  &__site-wide-nav
+    background: $color-dark
 
-      @include bp('portrait')
-        align-items: center
+    @include bp('landscape')
+      border-bottom: .2rem solid $color-white
 
-      @include bp('landscape')
+    &-menu
+
+      &-list
+        display: flex
         margin: 0 auto
-        max-width: 128.4rem
-        padding: $common-spacing * 2
+        max-width: 128rem
+
+        @include bp('landscape')
+          padding: 0 $common-spacing * 2
+
+        &-element
+          $list-element: &
+          display: none
+          width: 100%
+
+          @include bp('landscape')
+            display: block
+            flex-grow: 1
+            width: initial
+
+          &-link
+            display: block
+            padding: $common-spacing $common-spacing / 2
+            text-align: center
+
+            @include bp('landscape')
+              padding: $common-spacing
+
+          &-text
+            color: $color-light
+            font-weight: $font-weight-bold
+            font-size: 1.6rem
+            text-transform: uppercase
+
+            @include bp('large')
+              font-size: 1.8rem
+
+            &:hover
+              color: $color-white
+
+          &--selected
+            align-items: center
+            background-color: $color-main
+            display: flex
+            justify-content: flex-end
+            padding: 0 $common-spacing
+
+            @include bp('landscape')
+              display: block
+              padding: 0 $common-spacing / 2
+
+            #{$list-element}-text
+              color: $color-dark
+              font-size: 1.8rem
+
+              @include bp('landscape')
+                font-size: 1.6rem
+
+              @include bp('large')
+                font-size: 1.8rem
+
+  &__leg-nav
+    @extend .l-full
+    display: flex
+    justify-content: space-between
+    margin-bottom: 0
+    margin-top: 0
+    opacity: 1
+    padding: $common-spacing
+
+    @include bp('portrait')
+      align-items: center
+
+    .leg &
+      left: 0
+      position: absolute
+      right: 0
+      z-index: 3
+
+    .page-template-find-a-job &
+      position: relative
+
+    @include bp('landscape')
+      margin: 0 auto
+      max-width: 128.4rem
+      padding: $common-spacing * 2
 
     &-title // TODO change after merge with de-vue branch
```

Comment:
There's a `TODO` comment here. Is this still applicable?

Reply:
Removed -title sass.